src/main/java/uk/gov/hmcts/darts/arm/mapper/MediaArchiveRecordMapper.java

```diff
@@ -6,5 +6,5 @@
 import java.io.File;
 
 public interface MediaArchiveRecordMapper {
-    MediaArchiveRecord mapToMediaArchiveRecord(ExternalObjectDirectoryEntity externalObjectDirectory, File archiveRecordFile);
+    MediaArchiveRecord mapToMediaArchiveRecord(ExternalObjectDirectoryEntity externalObjectDirectory, File archiveRecordFile, String rawFilename);
```

Comment:
Not sure why the formatting is different here when compared to the other 3 files?